### DISCUSSION TITLE
添加一个选择显卡的功能

### DISCUSSION THREAD

**Discussion by fengweipeng** [Akegarasu/lora-scripts#277] [Ideas]
![image](https://github.com/Akegarasu/lora-scripts/assets/5653289/c2904bb6-9bf3-4eac-8aa0-a67cc9fcc5f4)
就像“绘世”中的那样，如果，电脑上有双显卡，可以选择要用那一张显卡。
现在，程序会使用系统默认的显卡进行训练。如果有双显卡，需要选择第二张显卡进行训练，就要手动修改：A启动脚本.bat
在
         set PYTHON=python\python.exe
后面添加一行：
        set CUDA_VISIBLE_DEVICES=1
变成：

```
@echo off
set HF_HOME=huggingface
set PYTHON=python
cd /d %~dp0
if exist ".\python\python.exe" (
        set PYTHON=python\python.exe
        _set CUDA_VISIBLE_DEVICES=1_
        echo 使用目录内的 python 进行启动....
) else (
    echo 尝试使用系统 python 进行启动....
)
%PYTHON% gui.py


pause
```

**Comment by onijack888** [comment]
请教秋叶大神，如果是需要两张显卡一起训练，丹炉里应该怎么设置呢？我选择了多显卡选项，但是训练直接报错。
